101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {    
+      "name": {
+          "type": "string",
+          "metadata": {
+              "description": "Name for the container group"
+          },
+          "defaultValue": "acinavcontainergroup"
+      },    
+      "navRelease": {
+          "type": "string",
+          "metadata": {
+              "description": "After the colon: Identifier for the NAV release like 2017-cu11-de or devpreview-finus (always keep 'microsoft/dynamics-nav:')"
+          },
+          "defaultValue": "microsoft/dynamics-nav:2017"
```

Comment:
is there an appropriate set of allowedValues here?

Reply:
That would be all of those: https://hub.docker.com/r/microsoft/dynamics-nav/tags/ and every month approx 15 are added, so that set would be quite big and changing every month

Reply:
Let's put that link in the description - so users know where to get a valid set of values for this

Reply:
done